packages/xo-server-test/src/vm/index.spec.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env jest */
+
+import { mapValues } from 'lodash'
+
+import config from '../_config'
+import xo from '../_xoConnection'
+
+describe('vm', () => {
+  describe('.set() :', () => {
+    const PROPERTIES_TO_SET_TO_NULL = {
+      coresPerSocket: 1,
+      cpuCap: 1,
+    }
+    it('set properties to null', async () => {
+      const id = await xo.createTempVm({
+        name_label: 'XO Test',
+        template: config.templates.default,
+        ...PROPERTIES_TO_SET_TO_NULL,
+      })
+      expect(xo.objects.all[id]).toMatchObject(PROPERTIES_TO_SET_TO_NULL)
+
+      await xo.call('vm.set', {
+        id,
+        ...mapValues(PROPERTIES_TO_SET_TO_NULL, _ => null),
+      })
+      await xo.waitObjectState(id, vm => {
+        Object.keys(PROPERTIES_TO_SET_TO_NULL).forEach(prop => {
+          expect(vm[prop]).toBe(undefined)
+        })
+      })
+    }, 6e3)
```

Comment:
Where does this value come from?

What does it mean?

Reply:
It's the test timeout.

Reply:
Why 6 seconds?

Reply:
The current test time execution is `4 seconds` but with some latency we can exceed `5 seconds`.

Reply:
Do you know why it's so long and what we could do to improve it?

Reply:
Does it now pass without timeout?

Reply:
Yes we don't need timeout anymore :+1: